### PR TITLE
CTCTRADERS-815 create consignor name page

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -262,6 +262,12 @@ isConsignorEoriKnown.heading = Do you know the consignor’s EORI number?
 isConsignorEoriKnown.checkYourAnswersLabel = Do you know the consignor’s EORI number?
 isConsignorEoriKnown.error.required = Select yes if you know the consignor’s EORI number
 
+consignorName.title = What is the consignor’s name?
+consignorName.heading = What is the consignor’s name?
+consignorName.checkYourAnswersLabel = What is the consignor’s name?
+consignorName.error.required = Enter the consignor'''s name
+consignorName.error.length = Consignor''s name must be 35 characters or less
+consignorName.error.invalid = Consignor''s name must only include letters a to z and numbers 0 to 9
```
Comment (chris-ianson) [inline conf/messages.en:270]: Double check the quotes here, should they be smart quotes (see lines 265 and 266)

File: app/forms/ConsignorNameFormProvider.scala [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+package forms
+
+import javax.inject.Inject
+import forms.mappings.Mappings
+import uk.gov.hmrc.play.mappers.StopOnFirstFail
+import play.api.data.Form
+
+class ConsignorNameFormProvider @Inject() extends Mappings {
+
+  val consignorNameRegex: String = "^[a-zA-Z0-9 ]*$"
```
Comment (chris-ianson) [inline app/forms/ConsignorNameFormProvider.scala:26]: You could use the \s flat to match any whitespace i.e: "^[a-zA-Z0-9\\s]*$"

**Review by chris-ianson — CHANGES_REQUESTED** [review]
Recommendation on regex and comment on messages file

File: app/forms/ConsignorNameFormProvider.scala (hunk 2 above)
Comment (kenneth-rayner) [reply app/forms/ConsignorNameFormProvider.scala:26]: Pastedthat in and it did not compile.So I left it as it was

Comment (chris-ianson) [reply app/forms/ConsignorNameFormProvider.scala:26]: Sorry Ken, my bad it actually needs a double backslash before the s "^[a-zA-Z0-9\\s]*$"

No problem, go with what you had

Comment (kenneth-rayner) [reply app/forms/ConsignorNameFormProvider.scala:26]: The one I have when I test it manually does allow spaces anywhere

**Pull request merged**